### DISCUSSION TITLE
No Spa Temp

### DISCUSSION THREAD

**Discussion by Exit2Studios** [tagyoureit/nodejs-poolController#519] [Q&A]
I hate asking seemingly stupid/basic questions, but I've searched and can't figure out why I have no Spa Temp showing, and therefore no ability to activate the heater.

<img width="401" alt="image" src="https://user-images.githubusercontent.com/16908875/177657046-742d15c7-df03-419a-90ca-d5370f05cd7f.png">

Any idea what I am missing?

**Comment by rstrouse** [comment]
You are missing a feed from a temperature sensor on waterSensor1 for a shared body system.  For a configuration like this where the two bodies share a filter, there is typically a single sensor that is for both bodies.  Where are you getting your Pool Temp?

**Comment by Exit2Studios** [comment]
I am getting my pool temp from a Pentair 10k thermosister, which is why I don't understand why it's not showing spa as well since I have the feed going from REM to njsPC and it's obviosly working when in Pool mode.

**Comment by rstrouse** [comment]
The feed from the generic sensor is going to the temps socket event with waterSensor1 as its property?

**Reply by Exit2Studios** [reply]
Yes, and is working for Pool Temp.

**Reply by rstrouse** [reply]
Hmmm do you see anything in the logs for REM.  Please upload a short replay.  It is very odd that it is not picking this up then.

**Reply by tagyoureit** [reply]
This is what you should see in the logs for REM.  REM doesn't care if the pool or spa is running but njsPC should parse that correctly.  I have my feed set to 5 samples hence the 5 values.

```
1|REM  | debug: Interpolating map thermistor10k input: 9258 {"low":{"input":9078,"value":81},"high":{"input":9526,"value":79}}
1|REM  | verbose: Feed evaluating waterSensor1: 80.28 samples [80.28,80.32,80.28,80.24,80.2]
1|REM  | verbose: Emitting: /temps : {"waterSensor1":80.28}
1|REM  | verbose: Feed sending waterSensor1: 80.28
1|REM  | verbose: Feed sending adcValue: 9.258
```
